src/obr/signac_wrapper/operations.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def print_operations(self):
-        ops = sorted(self.operations.keys())
+        ops = sorted(self.groups.keys())
```

Comment:
So does `.groups` also contain operations?

Reply:
I've checked. `groups` contains exactly the union of `operations` and the groups defined over `OpenFOAMProject.make_group()`.